docs/Systems/Tailwind/components/UtilityClassTemplate.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react"
+import { StoryWrapper } from "../../../../storybook/components/StoryWrapper"
+
+type Props = {
+  compiledCssPropertyName: string
+  classKeyValues: Array<{ utilityClassName; cssProperty }>
+  renderExampleComponent: (cssProperty: string) => React.ReactElement
+  isReversed?: boolean
+}
+
+export const UtilityClassTemplate = ({
+  compiledCssPropertyName,
+  classKeyValues,
+  renderExampleComponent,
+  isReversed = false,
+}: Props): React.ReactElement => (
+  <StoryWrapper hasRowDivider isReversed={isReversed}>
+    <StoryWrapper.RowHeader
+      headings={["Utility Class", "Compiled CSS", "Example"]}
+    />
+    {classKeyValues.map((presetData, index) => {
+      const { utilityClassName, cssProperty } = presetData
+
+      return (
+        <StoryWrapper.Row rowTitle="">
+          <p className="font-family-paragraph">{utilityClassName}</p>
```

Comment:
Maybe the `utilityClassName` to be a code font style as a small indication that this is the code that consumers need to copy

Reply:
Yeah that'd be nice but we don't have anything like that yet as far as I can tell. Let me see if I can find an easy way to do it

Reply:
Made a small `<CodeSnippet>` component for this. It copies the text to the clipboard on click.

![code-snippet-demo](https://user-images.githubusercontent.com/34709943/220510664-4800023a-9e71-4116-b853-246e8aa2931f.gif)